packages/lib/constants.ts

```diff
@@ -18,3 +18,12 @@ export const TRIAL_LIMIT_DAYS = 14;
 export const HOSTED_CAL_FEATURES = process.env.HOSTED_CAL_FEATURES || BASE_URL === "https://app.cal.com";
 /** @deprecated use `WEBAPP_URL` */
 export const NEXT_PUBLIC_BASE_URL = process.env.NEXT_PUBLIC_WEBAPP_URL || `https://${process.env.VERCEL_URL}`;
+export const LOGO = "/calendso-logo-white-word.svg";
+export const LOGO_ICON = "/cal-com-icon-white.svg";
+export const ROADMAP = `${WEBSITE_URL}/roadmap`;
```

Comment:
This is an external link so it shouldn't use `${WEBSITE_URL}`
```suggestion
export const ROADMAP = "https://cal.com/roadmap";
```